### DISCUSSION TITLE
Podman on Windows doesn't show the pull progress?

### DISCUSSION THREAD

**Discussion by Luxvao** [containers/podman#16712] [Q&A] [answered]
I've moved to running podman on Windows, but I'm missing the progress bar of pulling images. Is that just not avaliable on Windows?

**Comment by rhatdan** [comment, marked as answer]
This is a known issue in podman-remote (windows, mac) where we can not send the status bar over the API.

**Reply by Luxvao** [reply]
Oh okay

**Comment by bpstelios10** [comment]
In Podman Desktop you can follow the progress. Images -> Pull an image
